Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -268,6 +262,17 @@ def examineFiles(self,files):
     for loadable in loadables:
       loadable.files, distances, loadable.warning = DICOMUtils.getSortedImageFiles(loadable.files, self.epsilon)
 
+    # by default, prefer the all-files loadable
+    loadables[0].confidence = .55
```

Comment:
At this point, do we know that all loadables belong the same series and loadables[0] refers to the full series?

Reply:
Yes, that's the case.

Reply:
I agree though that relying on the order in the list could be fragile.  I'll save a dedicated variable pointing to the all-files loadable.

Reply:
Okay, this is now updated